### PR TITLE
[CARBONDATA-3211] Optimize the documentation

### PR DESCRIPTION
**Pull request by bbinwang**
Optimize the upper/lower case problem

 - [x] Any interfaces changed?
 NO
 - [x] Any backward compatibility impacted?
 NO
 - [x] Document update required?
NO
 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
  NO     
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
NO

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by bbinwang** [comment]
> Can one of the admins verify this patch?

are you machine？

**Comment by xubo245** [comment]
@bbinwang Please optimize the title and content of this PR. for example: [CARBONDATA-3226] Remove duplicated and useless files

File: datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ public LuceneFineGrainDataMapFactory(CarbonTable carbonTable, DataMapSchema data
           DataMapWriter.getDefaultDataMapPath(tableIdentifier.getTablePath(),
               segment.getSegmentNo(), dataMapName), segment.getConfiguration()));
     } catch (MemoryException e) {
-      LOGGER.error("failed to get lucene datamap , detail is {}" + e.getMessage());
+      LOGGER.error("failed to get lucene datamap, detail is {}" + e.getMessage());
```
Comment (qiuchenjian) [inline datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java]:
```suggestion
      LOGGER.error(String.format("failed to get lucene datamap, detail is %s", e.getMessage()));
```
I think this log can't print the exception message

**Comment by xubo245** [comment]
@bbinwang Please optimize the title to:[CARBONDATA-3211] Optimize the documentation

**Comment by bbinwang** [comment]
> @bbinwang Please optimize the title to:[CARBONDATA-3211] Optimize the documentation

ok

**Comment by xubo245** [comment]
add to whitelist

File: datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java (hunk 1 above)
Comment (xubo245) [reply datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java]: +1, it should can print the exception message, but String.format is better.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2178/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10434/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2393/

File: datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java (hunk 1 above)
Comment (bbinwang) [reply datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java]: fixed

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2202/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10459/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2419/

**Pull request closed without merging**